packages/addresses/package.json

```diff
@@ -36,6 +36,7 @@
         "compile:js": "tsup --config build-scripts/tsup.config.library.ts",
         "compile:typedefs": "tsc -p ./tsconfig.declarations.json",
         "dev": "jest -c node_modules/test-config/jest-dev.config.ts --rootDir . --watch",
+        "lint:fix": "pnpm eslint --fix src/* && pnpm prettier -w src/*",
```

Comment:
Running these serially kneecaps Turborepo from giving us MAX PARALLELISM. Make `test:lint:fix` and `test:prettier:fix` and then make a root task `"style:fix": "turbo run test:lint:fix test:prettier:fix"`

Reply:
I was going to suggest this too but what does turborepo do if you run two jobs in parallel that modify the same set of files? Is it smart enough to make that work without them ever colliding on the same file?

Reply:
I was about to write something something race conditions something something file locks but now that I think about it for two seconds, Prettier _has_ to run last in case Eslint has fixes.

Yeah OK ignore me parallelism doesn't make sense here.

Still prefer the term `"style:fix"` though because Prettier isn't _really_ lint.

Reply:
The ideal situation would be to get Prettier to run _only_ on changed files, but at this point I think we've probably collectively burned enough time on this.